src/drivers/dshot/DShot.cpp

```diff
@@ -489,27 +277,26 @@ DShotOutput::task_spawn(int argc, char *argv[])
 	return PX4_ERROR;
 }
 
-void
-DShotOutput::capture_trampoline(void *context, uint32_t chan_index,
-				hrt_abstime edge_time, uint32_t edge_state, uint32_t overflow)
+void DShot::capture_trampoline(void *context, const uint32_t channel_index, const hrt_abstime edge_time,
+			       const uint32_t edge_state, const uint32_t overflow)
 {
-	DShotOutput *dev = static_cast<DShotOutput *>(context);
-	dev->capture_callback(chan_index, edge_time, edge_state, overflow);
+	DShot *dev = static_cast<DShot *>(context);
+	dev->capture_callback(channel_index, edge_time, edge_state, overflow);
 }
 
-void
-DShotOutput::capture_callback(uint32_t chan_index,
-			      hrt_abstime edge_time, uint32_t edge_state, uint32_t overflow)
+void DShot::capture_callback(const uint32_t channel_index, const hrt_abstime edge_time,
+			     const uint32_t edge_state, const uint32_t overflow)
 {
-	fprintf(stdout, "DShot: Capture chan:%d time:%lld state:%d overflow:%d\n", chan_index, edge_time, edge_state, overflow);
+	fprintf(stdout, "DShot: Capture chan:%d time:%lld state:%d overflow:%d\n", channel_index, edge_time, edge_state,
+		overflow);
 }
 
-void
-DShotOutput::update_dshot_out_state(bool on)
+void DShot::update_dshot_out_state(const bool state)
```

Comment:
How is `state` clearer than `on`?

Reply:
what is the name of the method?

Reply:
so that this could be condensed: https://github.com/PX4/PX4-Autopilot/blob/25c113a5273ab315c511773b23ef81e2a93f0026/src/drivers/dshot/dshot.cpp#L1408

Reply:
That doesn't make it clearer, `state` alone could also refer to e.g. 3D mode, armed, etc (`on` was more specific). Yes this might be nitpicky, but as you want to improve it, please go all the way. `enable_outputs(bool enabled)` would be an option.

Reply:
Done! Thanks!